avm/res/insights/scheduled-query-rule/main.bicep

```
@@ -11,6 +11,9 @@ param location string = resourceGroup().location
 @description('Optional. The description of the scheduled query rule.')
 param alertDescription string = ''
 
+@description('Optional. The display name of the scheduled query rule.')
+param alertDisplayName string = ''
```

```suggestion
param alertDisplayName string?
```

Small optimization with the change suggested below 😉

---

Thank you.  Changed